scripts/setup-ubuntu.sh

```diff
@@ -65,12 +65,9 @@ PACKAGES+=" xmlto"
 PACKAGES+=" xmltoman"
 
 # Needed by python modules (e.g. asciinema) and some build systems.
-PACKAGES+=" python3.10"
-PACKAGES+=" python3.11"
 PACKAGES+=" python3-pip"
 PACKAGES+=" python3-setuptools"
```

Comment:
Besides, `python3.11-setuptools` is not provided by apt anymore, but it is necessary to build python-related packages.

It should be installed through pip in `termux_step_setup_python_pip`.

Reply:
> Besides, python3.11-setuptools is not provided by apt anymore, but it is necessary to build python-related packages.
>
> It should be installed through pip in termux_setup_python_pip.

There is no `python3.11-setuptools` package in ubuntu 22.04 either, right?

There is a single `python3-setuptools` package providing `/usr/lib/python3/dist-packages/setuptools` that is shared between `python 3.x` versions, both in ubuntu 22.04  and 24.04.

To clarify: The difference between ubuntu versions is that

- In ubuntu 22.04 the system `python3` is `python3.10`.
- In ubuntu 24.04 the system `python3` is `python3.12`.

For both ubuntu versions we install `python3.11` as an additional package, and in both ubuntu versions we have a single `python3-setuptools` package that provides `setuptools` for all `python3.x` versions.

Reply:
Thanks for looking into and reviewing this!

Reply:
OK. I don't know that `python3-setuptools` is shared between all `python3.x` packages.